DeveloperGuide.md.txt

```diff
@@ -0,0 +1,108 @@
+# Developer Guide
```

Comment:
Please make sure the PRs are coming from a clean branch -- I will not review unclean branches next time.
Clean branches meaning there should only be commits relating to the LO.